mne/preprocessing/ica.py

```diff
@@ -2566,29 +2568,22 @@ def corrmap(icas, template, threshold="auto", label=None, ch_type="eeg",
         template_fig.canvas.draw()
 
     # first run: use user-selected map
-    if isinstance(threshold, (int, float)):
-        if len(all_maps) == 0:
-            logger.info('No component detected using find_outliers.'
-                        ' Consider using threshold="auto"')
-            return icas
-        nt, mt, s, mx = _find_max_corrs(all_maps, target, threshold)
-    else:
-        paths = [_find_max_corrs(all_maps, target, t) for t in threshold]
-        # find iteration with highest avg correlation with target
-        nt, mt, s, mx = paths[np.argmax([path[2] for path in paths])]
+    threshold = np.atleast_1d(np.array(threshold, float)).ravel()
+    threshold_err = ('No component detected using find_outliers when '
+                     'using threshold%s %s, consider using a more lenient '
+                     'threshold' % (threshold_extra, threshold))
+    if len(all_maps) == 0:
```

Comment:
This is not just `if not all_maps`?

Reply:
Yes but I find this to be more explicit about what it's checking